packages/orbit-components/src/Stepper/types.d.ts

```diff
@@ -16,7 +16,7 @@ export interface SharedProps extends Common.Globals {
   readonly maxValue?: number;
   readonly minValue?: number;
   readonly ariaLabelValue?: string;
-  readonly ariaLabelledBy?: string;
+  readonly ariaDescribedby?: string;
```

Comment:
Why didn't you just add `ariaDescribedby` without removing `ariaLabelledBy`? I don't think it hurts to have it both and we can avoid having BC 🤔

Reply:
I was considering whether to remove `ariaLabelledBy` or not. My reasons for remove it were:

- we want to target the same text for all components (input, buttons) - in that case I've found unnecessary (and maybe misleading) to have both of these `aria-*` attributes.
- `aria Labelledby` replaces `aria Label` when it's filled in. `area Describedby` adds complementary text content to `aria-label`

Reply:
> we want to target the same text for all components (input, buttons)

Do you mean as an additional text? That could be true, but maybe someone wants to use `aria-labelledby` for labeling the input value as they already have usable element instead of using `aria-label`.